src/System.IO/src/System.IO.csproj

```diff
@@ -68,6 +68,7 @@
 
   <!-- Common -->
   <ItemGroup Condition="'$(TargetGroup)' == 'uap101aot'">
+    <Compile Include="$(CommonPath)\System\SystemException.cs" />
```

Comment:
From my understanding, that's what's supposed to happen. This, however, is just a temporary stub for `System.SystemException` for UWP. Issue https://github.com/dotnet/corefx/issues/12550 is tracking this.